components/layout-container/layout-container.js

```diff
@@ -0,0 +1,47 @@
+export default class LayoutContainer extends crsbinding.classes.BindableElement {
```

Comment:
I dont think this needs to be a bindable element its overkill

Reply:
I changed it to a HTMLElement.